stdlib/dynarray.ml

```diff
@@ -164,6 +164,11 @@ module Dummy : sig
         [Bytes.unsafe_to_string]), and avoids a copy of the input
         array when possible. *)
 
+    val unsafe_nocopy_get : ('a, _) with_dummy array -> 'a array
```

Comment:
It's a little bit misleading that a function called `nocopy` can copy its argument (I realize there is already another `nocopy` function with this property in this file).
Possibly  `unsafe_as_array_without_dummy`